stable/dogstatsd/values.yaml

```diff
@@ -0,0 +1,15 @@
+image:
+  repository: datadog/docker-dd-agent
+  tag: 11.0.5123-dogstatsd
+
+datadog:
+  apiKey: "Put your datadog API key here"
```

Comment:
I think `apiKey` should be commented out here and `NOTES.txt` can be used to stop the deployment of the Pods when this value is not specified. For reference take a look a https://github.com/kubernetes/charts/blob/master/stable/datadog/templates/NOTES.txt

Also I think a Secrets volume should be added to store sensitive data such as the API key